### ISSUE TITLE
Any plans to support SIMD instrinsics?

### ISSUE THREAD

**Issue by pragmascript** [beefytech/Beef#129]
If you need to leverage the hardware in tight loops, not being able to do SIMD is a show stopper for performance oriented code. LLVM autovectorization isn't bad, but does not work reliably enough, and if it does not, you are out of luck.

**Comment by bfiete** [comment]
Agreed. There are no solid plans yet for the API - submit requests/ideas if there are specific things you'd like to see. The general plan would be for very low-level access.

**Comment by pragmascript** [comment]
Yes, I think low-level access, like you have in C with `immintrin.h` would be preferable, I think. Building higher-level abstractions on top can be done later, if necessary.
